atom/renderer/api/lib/desktop-capturer.coffee

```diff
@@ -0,0 +1,10 @@
+ipc = require 'ipc'
+NativeImage = require 'native-image'
+
+getSources = (options, callback) ->
+  ipc.send 'ATOM_BROWSER_DESKTOP_CAPTURER_GET_SOURCES', options
+  ipc.once 'ATOM_REDNERER_DESKTOP_CAPTURER_RESULT', (sources) ->
```

Comment:
How often do you call the `getSources` API? It seems that the calling is too frequent. 
But I think it's safe, once the `getSources` gets result from main process, the event listener will get removed.

@zcbenz, looks like the behavior `EventEmitter.once` in electron is different than it in node. In node, this listener-limitted warning will not appear if I call `once` more than 10 times while it appears in electron.